modulo3/labeddit/README.md

```diff
@@ -1,68 +1,75 @@
-**Projeto LabeEddit**
+# Projeto LabEddit
 
 
 
-***Desenvolvedor***
+# 📄 Sobre
 
-Bruno Monteiro
+O projeto representa a implementação de uma rede social (nesse caso inspirada no Reddit), 100% funcional. O site foi desenvolvido utilizando ReactJS, seguindo o conceito de Mobile First e um layout pré-determinado.
 
+# 🔗 Link para acessar
 
-***O que funciona:***
+[keen-interest.surge.sh/](https://colorful-prose.surge.sh/login)
```

Comment:
aqui eu acho que tu pode por só o nome do projeto, porque assim, ele mostra um link e direciona pra outro (se o usuário for só copiar o texto, ele acaba indo pra outro projeto teu)